pages/thank-you.jsx

```diff
@@ -32,7 +32,7 @@ var ThankYou = React.createClass({
                 </h2>
               </div>
             </div>
-            {this.props.params && ['de', 'en-US'].indexOf(locale) !== -1 ? <Signup language={locale}/> : <Social language={locale}/> }
+            {this.props.params && ['de', 'en-US'].indexOf(locale) !== -1 ? <Signup locales={this.props.locales}/> : <Social language={locale}/> }
```

Comment:
We could also define this.props.language along with this.props.locales, but the above felt like the easiest fix.